aero.minova.core.application.system.service/src/main/java/aero/minova/core/application/system/controller/SqlProcedureController.java

```diff
@@ -271,6 +269,17 @@ public SqlProcedureResult calculateSqlProcedureResult(Table inputTable, List<Row
 								page);
 					}
 				}
+				// Dies muss ausgelesen werden, nachdem die ResultSet ausgelesen wurde, da sonst diese nicht abrufbar ist.
+				val returnCode = preparedStatement.getObject(1);
+				if (returnCode != null) {
+					int reCode = preparedStatement.getInt(1);
+					resultForThisRow.setReturnCode(reCode);
+					// Falls nicht alle ReturnCodes gleich sind, wird 1 als endgültiger ReturnCode rein geschrieben.
+					if (!result.getReturnCodes().isEmpty() && !result.getReturnCodes().contains(reCode) && result.getReturnCode() != 1) {
```

Comment:
Sollte hier nicht setwas wie  `if reCode != 0` stehen?
Wenn es einen Fehler gab, dann sollte der ReturnCode != 0 sein.

Reply:
> Sollte hier nicht setwas wie `if reCode != 0` stehen?
> Wenn es einen Fehler gab, dann sollte der ReturnCode != 0 sein.

Im Falle eines Fehlers wird in die Klasse 'ControllerExceptionHandler' gesprungen und nur eine Fehler-SqlProcedureResult geschickt, dessen ReturnCodes alle -1 sind.